src/calendar/header/style.js

```diff
@@ -32,6 +32,11 @@ export default function(theme={}) {
         },
         android: {
           tintColor: appStyle.arrowColor
+        },
+        web: {
```

Comment:
Where do you use this style?

Reply:
react-native-web

Reply:
Sorry, I don't get it, can you elaborate please

Reply:
There is a popular library called react-native-web that allows React Native apps to render in the web browser. This is officially supported by Expo. When the react-native-calendars library is rendered in the web browser, the arrows do not have any color. Instead, the arrows are invisible on the web. This is because there is a missing color style for the "web" platform. Adding this missing color style as I have done will allow the arrows to render on the web.

https://github.com/necolas/react-native-web

Reply:
Ok. Thanks. Can you please resolve conflicts so I can merge?

Reply:
Sorry for the delay. Done!